runtime/service_integ_test.go

```diff
@@ -351,71 +355,104 @@ func TestMultipleVMs_Isolated(t *testing.T) {
 				TimeoutSeconds: 300,
 			}
 
-			resp, err := fcClient.CreateVM(ctx, req)
-			require.NoError(t, err, "failed to create vm")
+			resp, createVMErr := fcClient.CreateVM(ctx, req)
+			if createVMErr != nil {
+				matches, err := findProcess(ctx, findFirecracker)
+				if err != nil {
+					return err
+				}
+				return fmt.Errorf(
+					"failed to create a VM while there are %d Firecracker processes: %w",
+					len(matches),
+					createVMErr,
+				)
+			}
 
-			var containerWg sync.WaitGroup
+			eg := &errgroup.Group{}
```

Comment:
We have `vmEg` above, does it make sense to call this `containerEg`?

Reply:
Yeah. Will rename the variable.